### DISCUSSION TITLE
tabella decodifica ansc_3

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#1138] [Supporto servizi cooperativi] [answered]
Al posto di questi due simboli (? nel rombo nero) cosa ci va?

![image](https://github.com/user-attachments/assets/a5e8eddf-d666-4c89-8597-084bd6cb6dac)

grazie

**Comment by annamaraspin** [comment]
si ripete anche nella versione 1.35

sollecito una risposta

**Comment by asqueglia** [comment, marked as answer]
Con il rilascio della versione 1.36.0, la segnalazione è stata corretta
